packages/cli/src/commands/preview-project/constants.ts

```diff
@@ -0,0 +1,23 @@
+import { Product } from './types';
+
+export const PRODUCT_PACKAGES = {
+  realm: '@redocly/realm',
+  'redoc-revel': '@redocly/redoc-revel',
+  'revel-reef': '@redocly/revel-reef',
+  'redoc-reef': '@redocly/redoc-reef',
+  redoc: '@redocly/redoc',
+  revel: '@redocly/revel',
+  reef: '@redocly/reef',
+};
+
+export const PRODUCT_NAMES: { [key in Product]: string } = {
+  redoc: 'Redoc',
```

Comment:
Since the command makes `redoc` available for preview, among other options, it looks like there's no need for the `preview-docs` command anymore. Are we going to deprecate it at some point? If yes, when do we want to add the deprecation message?

Reply:
This is a little confusing, but for now the `redoc` that can be previewed with the `preview-docs` command is not the current open source Redoc. The `preview-docs` command will stay relevant for a some more time.